factoring/factoring/embedding.py

```diff
@@ -55,4 +55,18 @@
                'carry2,1': [1484, 1492],
                'carry3,0': [1485, 1481],
                'sum1,1': [1741, 1739],
-               'sum2,1': [1489, 1495, 1487]}}
+               'sum2,1': [1489, 1495, 1487]},
+              'DW_2000Q_6':
+              {'and2,1': [590, 582, 450, 578],
+               'sum1,1': [469, 464, 592, 720],
+               'carry1,0': [727, 851, 723],
+               'and2,2': [577, 449, 455, 448],
+               'a1': [707, 711, 719], 'and2,0': [709, 717, 725],
+               'and0,1': [854, 843, 846], 'a0': [585, 841, 713],
+               'and0,2': [456, 597, 584, 589], 'carry2,1': [457, 454, 462],
+               'carry2,0': [722, 594, 466], 'sum2,1': [452, 467, 468, 460],
+               'a2': [710, 704, 576], 'b0': [840, 712, 718], 'b1': [706, 834, 842, 839, 847],
+               'b2': [579, 583, 591], 'and1,2': [715, 587, 459], 'carry3,0': [471, 463],
+               'and1,0': [835, 837, 853, 845], 'and1,1': [586, 724, 716, 714],
+               'carry1,1': [458, 453, 461]}
+              }
```

Comment:
if you put a trailing comma you'll avoid changing the line (like above) the next time we add an embedding.